content/en/tracing/profiler/enabling.md

```diff
@@ -60,22 +60,29 @@ The Datadog Profiler requires [JDK Flight Recorder][1]. The Datadog Profiler lib
 
 - Datadog strongly recommends that you specify the `service` and `version` as this gives you the ability to slice and dice your profiles across these dimensions. Use environment variables to set the parameters:
```

Comment:
Out of scope, but at some point we should collapse ^ into step 3 otherwise it's like "do this, do that, ok you're all done, just kidding! go back and change the thing you did before!" :-)

Reply:
Yes, let me do the change right after this PR.

Reply:
https://github.com/DataDog/documentation/pull/11144/files